en/docs/learn/configuring-account-disabling.md

```diff
@@ -0,0 +1,133 @@
+
+	1.	On the **Main** menu of the Management Console, click **Identity > Identity Providers > Resident**.
+
+		<img src="../../assets/img/learn/resident-menu-item.png" alt="Resident menu-item" width="200" style="border:1px solid grey">  
+
+	2.	Under the **Login Policies** section, click **Account Disabling**.
+
+		<img src="../../assets/img/learn/account-disabling-option.png" alt="Account Disabling Option" width="700" style="border:1px solid grey">  	
+
+	3.	Select the **Enable Account Disabling** check box. 
+
+		<img src="../../assets/img/learn/account-disabling-form.png" alt="Account Disabling form" width="600" style="border:1px solid grey">  	
+
+	4.	Click **Update**. 
+
+4.	To enable the account disabling claim:
+
+	1.	On the **Main** menu of the Management Console, click **Identity > Claims > List**.
+
+		<img src="../../assets/img/learn/claims-list-menu-item.png" alt="Claims List option" width="200" style="border:1px solid grey"> 	 		 		 
+
+	2.	Click **http://wso2.org/claims**.
+
+		<img src="../../assets/img/learn/claim-list-wso2.png" alt="WSO2 claim dialect" width="700" style="border:1px solid grey">  
+
+	3.	Under **Account Disabled**, click **Edit**.
+
+		<img src="../../assets/img/learn/account-disabled-edit.png" alt="Account Disabled claim edit option" width="700" style="border:1px solid grey">  
+
+	4.	Select **Supported by Default**.
+
+		<img src="../../assets/img/learn/account-disabled-claim-supported-by-default.png" alt="Account Disabled claim's Suppported by Default option" width="700" style="border:1px solid grey">  
+
+	5. Click **Update**.
+
+## Try out
+
+1.	To create the user account for Larry:
+
+	1.	On the **Main** menu of the Management Console, click **Identity > Users and Roles > Add**.
+
+		<img src="../../assets/img/learn/add-users-and-roles-menu-item.png" alt="Add Users and Roles menu-item" width="200" style="border:1px solid grey">  
+
+	2.	Click **Add New User**.
+
+		<img src="../../assets/img/learn/add-new-user-option.png" alt="Add New User option" width="700" style="border:1px solid grey">  
+
+	3.	Enter the required data as follows.
+
+		<img src="../../assets/img/learn/add-new-user-screen.png" alt="Add New User screen" width="700" style="border:1px solid grey">   
+
+		-	**Domain**: `Primary`
+		-	**Username**: `Larry`
+
+	4.	Click **Finish**. 
+
+2.	To assign login permissions to the user:
+	
+	1.	Click the **View Roles** option of Larry.
+
+		<img src="../../assets/img/learn/view-roles-option.png" alt="View Roles option" width="700" style="border:1px solid grey"> 	 		 		
+
+	2.	Click **Permissions**.
+
+		<img src="../../assets/img/learn/role-permissions-option.png" alt="Role Permissions option" width="700" style="border:1px solid grey"> 	 		 		
+
+	3.	Select **Login** and click **Update**.
+
+		<img src="../../assets/img/learn/login-permission.png" alt="Login permission" width="700" style="border:1px solid grey">  
+
+3.	To disable Larry's user account:
+	
+	1.	Click **User Profile** option of Larry.
```

Comment:
Shall we remove ".. of Larry"?

Reply:
This is required as there will be other users that have the User Profile button. (These are user rows)